airbyte-webapp/scripts/license-check.js

```diff
@@ -82,39 +82,30 @@ function validateLicenes(licenses, allowedLicenes, usedOverwrites) {
       console.log(`Guessed license for package ${pkg}: ${license}`);
     }
     if (!license || !allowedLicenes.includes(license)) {
-      licensesValid = false;
-      console.error(`Package ${pkg} has incompatible license: ${license}`);
+      if (pkg !== "backslash@0.2.0") {
```

Comment:
We should not add any checks in this code here. This script already has a way to overwrite licenses for specific packages. Please add a license overwrite on top to the `LICENSE_OVERWRITES`, that will state the license this package had in that version (which is according to its [README](https://github.com/Qix-/node-backslash/tree/0.2.0)) the MIT license. If a license actually would not have any licensed specified for the version we'd be pulling in, we could not use it in our code!

Reply:
I had some comments that missed getting posted. Moved this code to the appropriate place.
I had confirmed it was using MIT.